src/modules/meteor/assets/templates/start.sh

```diff
@@ -54,57 +54,68 @@ echo "Ran <%= docker.image %>"
 sleep 15s
 
 <% if(typeof sslConfig === "object")  { %>
+
   <% if(typeof sslConfig.autogenerate === "object")  { %>
-    echo "Running autogenerate"
+    # If using let's encrypt pull and setup the companion proxy
+    echo "Running Autogenerate Proxy"
     # Get the nginx template for nginx-gen
     wget https://raw.githubusercontent.com/jwilder/nginx-proxy/master/nginx.tmpl -O /opt/$APPNAME/config/nginx.tmpl
-    
-    # Update nginx config based on user input or default passed by js
-sudo cat <<EOT > /opt/$APPNAME/config/nginx-default.conf
-client_max_body_size $CLIENTSIZE;
```

Comment:
ejs doesn't like undefined variables. Please remove `$CLIENTSIZE` from the top of the file.